charts/pip-account-management/values.stg.template.yaml

```diff
@@ -2,3 +2,5 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  environment:
```

Comment:
Similar comment to other PR regarding sds flux

Reply:
Have created the flux PR:
https://github.com/hmcts/sds-flux-config/pull/3134/files